React/Fabric/Mounting/ComponentViews/Text/RCTParagraphComponentView.mm

```diff
@@ -100,9 +100,29 @@ - (void)updateProps:(Props::Shared const &)props oldProps:(Props::Shared const &
 - (void)updateState:(State::Shared const &)state oldState:(State::Shared const &)oldState
 {
   _state = std::static_pointer_cast<ParagraphShadowNode::ConcreteState const>(state);
+  auto const &paragraphProps = *std::static_pointer_cast<ParagraphProps const>(_props);
+  if (_state && paragraphProps.accessible) {
+    auto &data = _state->getData();
+    if (![_accessibilityProvider isUpToDate:data.attributedString]) {
+      auto const &accessibilityProps = *std::static_pointer_cast<AccessibilityProps const>(_props);
+      BOOL accessibilityLiveRegionEnabled = accessibilityProps.accessibilityLiveRegion != AccessibilityLiveRegion::None;
+      if (accessibilityLiveRegionEnabled) {
+        self.triggerLiveRegionAccessibilityAnnouncement = YES;
+      }
+    }
+  }
   [self setNeedsDisplay];
 }
 
+- (void)finalizeUpdates:(RNComponentViewUpdateMask)updateMask
+{
+  if (self.triggerLiveRegionAccessibilityAnnouncement) {
+    [self announceForAccessibilityWithOptions:self.accessibilityLabel];
```

Comment:
does not announce other changes (for ex. accessibilityState) because over-rides the parent logic without calling super